src/shared/AsyncTypeaheadSelect/AsyncTypeaheadSelect.tsx

```diff
@@ -47,6 +51,12 @@ export const AsyncTypeaheadSelect: VFC<AsyncTypeaheadSelectProps> = ({
     undefined
   );
 
+  useEffect(() => {
+    if (submitted && required)
+      setValidation({ isValid: false, message: t("common:required") });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Never shut down this error, it's a hint that something is fishy in your logic.
In this case, you don't need an effect at all, the validation can be derived by combining the temporary one of the typeahead - in the local state - and the one coming from the props.